babi/file.py

```diff
@@ -548,9 +557,12 @@ def tab(self, margin: Margin) -> None:
             self._tab(margin)
 
     def _dedent_line(self, s: str) -> int:
-        bound = min(len(s), self.buf.tab_size)
+        tab_size = 1
+        if self.buf.expandtabs:
+            tab_size = self.buf.tab_size
```

Comment:
isn't this just `tab_size = len(self.buf.tab_string)`

Reply:
Oops, I missed that :thinking: